R/accounts.R

```diff
@@ -484,17 +484,21 @@ resolveAccount <- function(account, server = NULL) {
   }
 }
 
-isShinyapps <- function(server) {
+isCloudServer <- function(server) {
   identical(server, "shinyapps.io") | identical(server, "rstudio.cloud")
```

Comment:
Should we prefetch the rebranding work and include "posit.cloud" here?

Reply:
There is a separate PR for rebranding: https://github.com/rstudio/rsconnect/pull/614

Reply:
Nit: You wanted `||` here, rather than `|`. The two-character form always returns a length-one vector.
https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/Logic